### ISSUE TITLE
Editing Spatial Layouts through Tactile Templates for People with Visual Impairments

### ISSUE THREAD

**Issue by Wotipati** [Wotipati/Blind_Accessibility_Papers#6]
### Links
- Paper: https://dl.acm.org/citation.cfm?id=3300436
- Movie: https://www.youtube.com/watch?v=ig3DRfxDQKI

### Abstract
- Spatial layout is a key component in graphic design. While people who are blind or visually impaired (BVI) can use screen readers or magnifiers to access digital content, these tools fail to fully communicate the content's graphic design information. Through semi-structured interviews and contextual inquiries, we identify the lack of this information and feedback as major challenges in understanding and editing layouts. Guided by these insights and a co-design process with a blind hobbyist web developer, we developed an interactive, multimodal authoring tool that lets blind people understand spatial relationships between elements and modify layout templates. Our tool automatically generates tactile print-outs of a web page's layout, which users overlay on top of a tablet that runs our self-voicing digital design tool. We conclude with design considerations grounded in user feedback for improving the accessibility of spatially encoded information and developing tools for BVI authors.
<img width="1008" alt="スクリーンショット 2019-10-27 20 59 18" src="https://user-images.githubusercontent.com/26996041/67634248-aed2ac00-f8fc-11e9-9697-dc969034fac0.png">

スライド出典: [CHI勉強会2019](https://sigchi.jp/seminar/chi2019/)
スライド製作者: 北海道大学 渡邉拓貴さん
